bin/prettier.js

```diff
@@ -283,6 +288,9 @@ if (
       "  --list-different or -l   Print filenames of files that are different from Prettier formatting.\n" +
       "  --config                 Path to a prettier configuration file (.prettierrc, package.json, prettier.config.js).\n" +
       "  --resolve-config <path>  Resolve the path to a configuration file for a given input file.\n" +
+      "  --ignore-path <path|string>  \n" +
```

Comment:
@azz your can use `/path/to/.ignored` (path) or `.ignored` (string), but we can simplify just use `<path>`